### PR TITLE
add cookie local file

### PR DESCRIPTION
**Pull request by dgkang**

### CODE REVIEW

**Comment by yihong0618** [comment]
can you mention this cookie way in `README
`

**Comment by yihong0618** [comment]
@dgkang it seems that we add the as an option maybe better.
And could you take a look at #9  I think its another way to access the `cookie-file` maybe easier?

**Comment by dgkang** [comment]
README 已做修改，增加从本地文件加载cookie，是因为，在Windows Shell的命令行参数老是解析cookie失败，所以增加了该功能

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Download all your kindle books script.
 2. visit https://www.amazon.com/hz/mycd/myx#/home/content/booksAll/dateDsc/ 
 3. find cookie F12 XHR or other ways
 4. right click this page source then find `csrfToken` value copy
-5. run: `python3 kindle.py ${csrfToken} --is-cn --cookie {cookie} --cookie-file {cookie file path}`
+5. run: `python3 kindle.py ${csrfToken} --cookie {cookie} --cookie-file {cookie file path}`
```
[the remark this answers is not in the document]
Comment (dgkang) [reply README.md:21]: 嗯，是的，没错

File: README.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ Download all your kindle books script.
 2. 访问 https://www.amazon.cn/hz/mycd/myx#/home/content/booksAll/dateDsc/
 3. 找到 cookie XHR 或者其他的方式
 4. 右键查看源码，搜索 `csrfToken` 复制后面的 value
-5. 执行 `python3 kindle.py ${cookie} ${csrfToken} --is-cn`
+5. 执行 `python3 kindle.py  ${csrfToken} --is-cn --cookie {cookie} --cookie-file {cookie file path}`
```
Comment (frostming) [inline README.md:14]:
```suggestion
5. 执行 `python3 kindle.py  ${csrfToken} --is-cn --cookie {cookie}` 或 `python3 kindle.py  ${csrfToken} --is-cn --cookie-file {cookie file path}`
```

File: README.md [hunk 3]
```diff
@@ -11,14 +11,14 @@ Download all your kindle books script.
...
 
 # how to `amazon.com`
 1. login amazon.com
 2. visit https://www.amazon.com/hz/mycd/myx#/home/content/booksAll/dateDsc/ 
 3. find cookie F12 XHR or other ways
 4. right click this page source then find `csrfToken` value copy
-5. run: `python3 kindle.py ${cookie} ${csrfToken}`
+5. run: `python3 kindle.py ${csrfToken} --cookie {cookie} --cookie-file {cookie file path}`
```
Comment (frostming) [inline README.md:21]:
```suggestion
5. run: `python3 kindle.py ${csrfToken} --cookie {cookie}` or. `python3 kindle.py ${csrfToken} --cookie-file {cookie file path}`
```

**Pull request merged**